setup.py

```diff
@@ -63,7 +63,7 @@ def apply_patches(self, base_dir):
 
     packages=['motioneye'],
 
-    install_requires=['tornado>=3.1,<6', 'jinja2', 'pillow', 'pycurl'],
+    install_requires=['tornado>=3.1,<6', 'jinja2', 'pillow', 'pycurl', 'webdavclient3==0.12'],
```

Comment:
Curious if webdav provides REST api so as to avoid adding webdavclient here?

Reply:
Actually I used only 3 or so out of many webdav calls. Maybe the whole functionality could also be done by building the requests manually with pycurl or requests. Should i take a look into this? Still some answers of the webdav server might be in XML which has to be parsed. Also the webdav3 library handles errors nicely and throws meaningful exceptions.

Reply:
If a new upload service is doable without adding additional library, I would think that's the way to go to keep this project lightweight.

Reply:
I certainly don't want motionEye to pull in a dep like `webdavclient`. This should be optional.